src/components/BooksQuery.tsx

```diff
@@ -1,67 +1,59 @@
-import axios from "axios";
-import { useState } from "react";
-import Card from "@mui/material/Card";
+import { SyntheticEvent, useState } from "react";
 import "./books.css";
-import { CardContent, Typography } from "@mui/material";
-type book = {
-  id: number;
-  title: string;
-  author: string;
-  year: string;
-};
+import { useGoogleBooksApi } from "../hooks/useGoogleApi";
 const BooksQuery = () => {
   const [book, setBooks] = useState("");
-  const [list, setList] = useState([]);
-  const submit = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+  const [{ response, isLoading, isError }, searchBooks] =
```

Comment:
On the first load, I want to see a search for something. For example "react" => `useGoogleBooksApi("react");`